src/views/tutorial-view/components/collection-card/types.ts

```diff
@@ -0,0 +1,11 @@
+import { Collection as ClientCollection } from 'lib/learn-client/types'
+
+export interface CollectionCardProps
+  extends Pick<
+    ClientCollection,
+    'id' | 'name' | 'slug' | 'theme' | 'description'
+  > {
+  /** Optional class name to pass to the root element, which is expected to be the underlying CardLink component. */
+  className?: string
+  numTutorials: number
```

Comment:
Formatting request - feel free to not apply this suggestion via GitHub PR UI and do it manually if that's preferable! 💖 

```suggestion
  /**
   * Optional class name to pass to the root element, which is expected to be
   * the underlying CardLink component.
   */
  className?: string

  /**
   * TODO: add description
   */
  numTutorials: number
```